common/religions/primitive_group.txt

```diff
@@ -370,10 +370,84 @@ primitive_group = {
 		# Theocratic laws
 		priests_can_inherit = no
 		priests_can_marry = no
-
+		
+		max_consorts = 3
+		women_can_take_consorts = yes
+		men_can_take_consorts = yes
+		
 		# Gender laws
 		matrilineal_marriages = yes
 
 		dynamic_cult = no	# Warcraft - can't handle it yet
 	}
+	
+	harpy_religion = {
+		graphical_culture = mongolgfx
+		secondary_event_pictures = shamanism
+
+		icon = 91
+		heresy_icon = 34
+
+		color = { 255 182 193 }
+
+		scripture_name = THE_STORIES
+		priest_title = SHAMAN
+
+		high_god_name = SPIRITS
+
+		god_names = {
+			SPIRITS
+		}
+
+		evil_god_names = {
+			EVIL_SPIRITS
+		}
+
+		# Intermarries
+		# intermarry = shamanism_group
+
+		# Modifiers
+		character_modifier = {
+			# Bonuses
+			intrigue = 1
+			female_opinion = 10
+			
+			# Maluses
+			diplomacy = -1
+			male_opinion = -25
+			
+			# Opinions of other religions
+			burning_legion_religion_opinion = -200
+			death_god_opinion = -200
+			old_gods_worship_opinion  = -200
+		}
+
+		# Secular laws
+		can_excommunicate = yes
+		max_consorts = 5
+		women_can_take_consorts = yes
+		men_can_take_consorts = no
+		hard_to_convert = yes
+
+		# War laws
+		allow_looting = yes
+		can_call_crusade = no
+		can_grant_claim = yes
+		can_grant_invasion_cb = invasion
+		peace_prestige_loss = yes
+		raised_vassal_opinion_loss = no
+
+		# Theocratic laws
+		priests_can_inherit = no
+		priests_can_marry = no
+
+		
+		# Gender laws
+		matrilineal_marriages = yes
+		male_temple_holders = no
```

Comment:
Sadly, it's not working good without decent amount of work spent on succesion types and some events. I was trying to do it on https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/tree/gender-laws-overhaul , but work has been frozen.

So, comment it out...
```suggestion
		# male_temple_holders = no
```